nasa-gibs/worldview
# WV-3226 Look and Feel Changes

## Description
This update changes the style and look of some things on the WV landing screen, including font, color, icons, and the tour story boxes. See the design [here](https://bugs.earthdata.nasa.gov/secure/attachment/103121/wv_hds_comp_3a.png). Note the eye icons in the design are intentionally being ignored in favor of our current eye icons.

## How To Test
1. `git checkout wv-3226-look-and-feel`
2. `npm ci`
3. `npm run watch`
4. Open a fresh instance of Worldview
5. Confirm that many things on the front page matches the designs
6. Confirm that functionality of basic features still works
